extensions/amp-experiment/0.1/amp-experiment.js

```diff
@@ -46,12 +47,12 @@ export class AmpExperiment extends AMP.BaseElement {
         Object.keys(config).map(experimentName => {
           return allocateVariant(this.getWin(), config[experimentName])
               .then(variantName => {
-                if (variantName) {
-                  results[experimentName] = variantName;
-                }
+                results[experimentName] = variantName;
```

Comment:
Very legit questioning. It's indeed a bit overwhelming here. Let me break some inlining to gain some readability :-)